### ISSUE TITLE
Team Points

### ISSUE THREAD

**Issue by dbr96** [kmb266/cs5150-basketball-project#2]
https://github.com/kmb266/cs5150-basketball-project/blob/37a68d206d91ba7208e61c851fe2e65c8e60493b/app/advanced_stat.py#L4

I assume this is pulling Team Points from somewhere else in the program since there is no forumla?

**Comment by kmb266** [comment]
It is not pulling from anywhere that I can see. Do you have a formula for team points?

**Comment by dbr96** [comment]
No. There is no formula for Team Points. It would literally just be the total number of points from the box score. I guess it would be from the XML file in that case. My point with this question is when a formula has Team Points as part of the equation, where is the program getting the number from? Hope that made sense.
